worker/deploy/hosts.conf

```diff
@@ -1,9 +1,9 @@
+# For each worker node write a field started by `worker_ip`
```

Comment:
is it really *anything*? or it is a number ranging from 1 until N, where N is the number of workers nodes to be deployed

Reply:
It is really anything.